README.md

```diff
@@ -60,16 +60,19 @@ uds deploy k3d-core-demo:0.16.1
 
 #### UDS Package Development
 
-In addition to the demo bundle, a [k3d-core-istio-dev bundle](./bundles/k3d-istio/README.md) also exists to work with UDS Core with only Istio & Pepr installed. Run the command below to use it:
+In addition to the demo bundle, a [k3d-core-istio-dev bundle](./bundles/k3d-istio/README.md) also exists to work with UDS Core with only Istio, Keycloak & Pepr installed. Run one of the commands below to use it.
```

Comment:
```suggestion
In addition to the demo bundle, a [k3d-slim-dev bundle](./bundles/k3d-slim-dev/README.md) also exists to work with UDS Core with only Istio, Keycloak & Pepr installed. Run the command below to use it.
```

Reply:
thank i knew i was going to miss something. Will update.